<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="tittle_app">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SolatApp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_kota">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Kota</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sound">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Suara Notifikasi</string>
    </property>
   </widget>
   <widget class="QComboBox" name="list_city">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sound_path">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sound_path">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>160</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QLabel" name="clock">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>time_run</string>
    </property>
   </widget>
   <widget class="QLabel" name="parameter_city">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="route_city">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="distance_city">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="penetapan_shubuh">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="penetapan_ashr">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="penetapan_isya">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="penetapan_imsyak">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>420</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="jadwal">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>440</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INFO: Update secara berkala tiap bulan!</string>
    </property>
   </widget>
   <widget class="QLabel" name="bottom_label2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>550</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data diambil dari</string>
    </property>
   </widget>
   <widget class="QLabel" name="link2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;a href=&quot;http://jadwalsholat.org/&quot;&gt;jadwalsholat.org&lt;/a&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>471</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="table">
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>55</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>56</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>15</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>15</number>
       </attribute>
       <column>
        <property name="text">
         <string>Imsyak</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Shubuh</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Terbit</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Dhuha</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Dzuhur</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ashr</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Maghrib</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Isya</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="compass_image">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>260</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_button">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>210</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="last_update">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
